modules/prebidServerBidAdapter/index.js

```diff
@@ -523,6 +523,14 @@ const OPEN_RTB_PROTOCOL = {
       request.ext.prebid = Object.assign(request.ext.prebid, _s2sConfig.extPrebid);
     }
 
+    /**
+     * @type {(string[]|undefined)} - OpenRTB property 'cur', currencies available for bids
+     */
+    const adServerCur = config.getConfig('currency.adServerCurrency');
+    if (Array.isArray(adServerCur) && adServerCur.length) {
```

Comment:
The official documentation lists `adServerCurrency` as a string: http://prebid.org/dev-docs/modules/currency.html

So wouldn't this never be true?

Reply:
Rich is right -- the source here (getConfig) is a string while the destination (OpenRTB) is an array.

Reply:
So I guess we either wrap our always singular value in an array or update the `adServerCurrency` API?

Reply:
Well we're not going to do the latter in the short term. How about a compromise -- if the value is a string, wrap it an array. If it's an array, copy the first element. @idettman ?

Reply:
Updated code to support `string` or `array` values for `adServerCurrency`